lib/osf-components/addon/components/editable-field/template.hbs

```diff
@@ -1,74 +1,78 @@
-<this.managerComponent as |manager|>
-    {{#if manager.shouldShowField}}
-        <div local-class='Editable'>
+{{#if @manager.shouldShowField}}
+    <div data-test-editable-field={{@name}}>
+        <div local-class='Editable__header'>
             <h4>{{@title}}</h4>
-            {{#if manager.userCanEdit}}
+            {{#if @manager.userCanEdit}}
                 <OsfButton
                    data-test-edit-button={{@name}}
                    aria-label={{t 'general.edit'}}
                    data-analytics-name='Edit'
                    local-class='EditButton'
                    @type='link'
-                   @onClick={{action manager.startEditing}}
+                   @onClick={{action @manager.startEditing}}
                 >
                     <FaIcon @icon='pencil' @fixedWidth={{true}} />
                 </OsfButton>
             {{/if}}
         </div>
-        {{#if manager.fieldIsEmpty}}
-            {{#if manager.userCanEdit}}
-                {{manager.emptyFieldText}}
-            {{/if}}
-        {{else}}
-            {{yield (hash
-                display=(component 'editable-field/block')
-                manager=manager
-            )}}
+    </div>
+    {{#if @manager.fieldIsEmpty}}
+        {{#if @manager.userCanEdit}}
+            {{@manager.emptyFieldText}}
         {{/if}}
+    {{else}}
+        {{yield (hash
+            display=(component 'editable-field/block')
+            manager=@manager
+        )}}
     {{/if}}
+{{/if}}
 
-    <BsModal
-        @open={{manager.inEditMode}}
-        @onHide={{action manager.cancel}}
-        as |modal|
-    >
-        <modal.header>
-            <h3 class='modal-title'>
+<BsModal
+    @open={{@manager.inEditMode}}
+    @onHidden={{@manager.cancel}}
```

Comment:
Could you add `@backdropClose={{false}}` here? Since they're actually doing stuff in these modals we want them to be harder to close accidentally.